### PR TITLE
Archer Pan HW1

### PR DESCRIPTION
**Pull request by yuepan4**
For the homework, I went to the hw01 Github page of this class to fork the file to my Github account. Then I clicked the green "clone or download" green button to get the URL, then I opened my Rstudio--File--New project--version control--git, and I pasted the URL here to make sure that my local Rstudio is paired with my Github account so that I can pull and push commits. I went through the manual of Markdown for R and Github to make sure that I understand how to format my text, insert picture, links and so on (My understanding of these codes are reflected in my homework). For every commit I made, on my local R studio, I first clicked save, then checked the "staged" box in front of "ReadMe", then commit with reminder notes in the box, then push, then pull, after this I was able to view how my codes look like on Github. I repeat this process until I got the ideal version I want. After all this, I came back to create this pull request to submit my homework. 
The Final Version#1 is my final version of my homework.

### CODE REVIEW

**Comment by jmausolf** [comment]
## Evaluation HW-01 for Archer Pan

| Topic | Excellent | Satisfactory | Needs Work |
| --- | --- | --- | --- |
| **Coding style** |        x  |     |             |
| **Coding strategy** |     x   |        |        |
| **Presentation: graphs** |      |     |       |
| **Presentation: tables** |       |       |     |
| **Achievement, creativity** |x      |      |         |
| **Ease of access** | x |                 |             |

Remarks:
- For the header of the assignment, you probably could have removed the "hw01" header and subsequent lines with the assignment details. If you would like to include these, they could probably go in your pull request title/notes.
- Instead of inserting a link to the picture, you could embed it inline. That is:
```
![My dog](http://ww4.sinaimg.cn/mw690/7f6853e5gw1fbfm4ziuj0j20qo0zkn2z.jpg)
```

**Above code produces this:**

My dog

- Otherwise, you effectively used four of the five style elements, italics, headers, list, and link. Good job.
- Great use of git. Proved you can do the workflow.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+Due Date:Jan 11th    
+## **Archer Pan's Brief Biography**
+###**Education**
+I'm currently a MAPSS student concentrating on *Behavioral Science*.I'm researching false memeory and foreign language effect under supervision of Dr.Boaz Keysar.  
+Here is the link to the lab (https://mdl.uchicago.edu/directory/boaz-keysar)  
+Before coming to Uchicago, I received my B.A from University of Illinois at Urbana-Champaign.
+####**Pet**
+I'm a dog lover, I have a five-month-old mininature goldendoodle.  
+Here is a pircture of him:
+http://ww4.sinaimg.cn/mw690/7f6853e5gw1fbfm4ziuj0j20qo0zkn2z.jpg
```
Comment (longxuan0908) [inline README.md:15]: What you want to do here might be to embed the image into the markdown file as pointed out by the other peer reviewer

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
 # hw01
 Repository for first CFSS homework assignment
+
+Name:Archer Pan  
+File:HW1  
+Due Date:Jan 11th    
+## **Archer Pan's Brief Biography**
+###**Education**
+I'm currently a MAPSS student concentrating on *Behavioral Science*.I'm researching false memeory and foreign language effect under supervision of Dr.Boaz Keysar.
```
Comment (longxuan0908) [inline README.md:9]: You do not have to italicize "behavioral science" when you are typing the code, the two asterisks will take care of it.

**Comment by longxuan0908** [comment]
In general you did a good job at using headers, links, lists, as well as using italics. You also successfully posted this homework on github which means you know how to pull, push, and commit. Overall, good job.

File: README.md (hunk 2 above)
Comment (longxuan0908) [reply README.md:9]: Wait, it might be just an auto formatting by this website. If that is the case, don't mind.

**Pull request closed without merging**